### DISCUSSION TITLE
article/c543963f-402f-4901-8aa4-15d6fe07e8e7

### DISCUSSION THREAD

**Discussion by giscus** [ymh0000123/boke#10] [博客评论]
# article/c543963f-402f-4901-8aa4-15d6fe07e8e7

使用Cloudflare和GitHub搭建服务速度快并且免费的图床

https://xiaofeishu-boke.netlify.app/article/c543963f-402f-4901-8aa4-15d6fe07e8e7

**Comment by LineXic** [comment]
很棒的项目，但有个问题，你能告诉我怎么让图片在主页显示吗

**Reply by ymh0000123** [reply]
哪个主页

**Reply by LineXic** [reply]
这个
![图片](https://img.linexic.top/file/1d5354de07d215dd4e9c2.jpg)
麻烦啦

**Reply by ymh0000123** [reply]
> 这个
> ![图片](https://img.linexic.top/file/1d5354de07d215dd4e9c2.jpg)
> 麻烦啦

这个是每次更新运行博客提到的py脚本 可以放在构建命令里面

**Reply by LineXic** [reply]
GitHub Actions？还是直接把你文章里的py弄到根目录

**Reply by ymh0000123** [reply]
> GitHub Actions？还是直接把你文章里的py弄到根目录

我的做法是在cf部署的地方运行py脚本 没用GitHub Actions

**Reply by LineXic** [reply]
好的，我在去捣鼓捣鼓，麻烦你了
